_posts/2024-09-17-pyopensci-sustainability-plan-2024.md

```diff
@@ -0,0 +1,171 @@
+
+* continue supporting the vibrant pyOpenSci community, and
+* expand our programs specifically to develop education resources that teach open science, reproducibility, and packaging topics.
+
+
+### How pyOpenSci allocated funds to support efficient growth
+
+I like to think of pyOpenSc as a scrappy, efficient organization. Scrappy means we are deeply committed to managing our resources as efficiently as possible. We have a small core paid team that optimizes our time and online infrastructure to reduce costs and maximize impact.
+
+We minimize overhead costs by:
+
+* leveraging & building open source tools,
+* focusing funds on automating workflows using tools like CI (continuous integration) to reduce manual effort, and
+* prioritizing support for essential, high-impact activities that drive our mission forward.
+
+Looking to the future, we will only grow our core team as it aligns with our core budget. This strategic approach enables pyOpenSci to maintain the quality and reach of pyOpenSci programs while building a sustainable future.
+
+### Other non profits have done this well
+
+I am inspired by other organizations that have found their sustainability balance. Organizations such as NumFocus and the Python Software Foundation use events such as:
+
+* SciPy meetings,
+* pyData meetings, and
+* PyCon US
+
+as core elements of their sustainability models. Community-driven events allow these organizations to build community while funding their core mission, vision, and values.
+
+I am also looking to the membership model that the Carpentries has successfully employed to build core sustainability for their team.
+
+pyOpenSci will experiment to find the balance between freedom, openness, and sustainability in our training program. I'm confident that we will succeed.
+
+## What's next for pyOpenSci - The Road Ahead
+
+Our next training event is the inaugural [pyOpenSci Fall Festival](events/pyopensci-2024-fall-festival.html)--a week-long event that teaches skills needed to:
+
+* write cleaner, more modular code,
+* package and share code,
+* publish and cite code, and
+* create reproducible reports that connect code, data, and outputs into a dynamically produced interactive publication.
+
+During these workshops we will explore the use (and dangers) of using LLM's in the learning and workflow development process. The first day is a set of free, online keynote talks. Be sure to get your ticket before they sell out!
+
+I look forward to seeing you there!
+
+<div class="notice" markdown="1">
+## Get involved with pyOpenSci
+
+* support a scholarship (or two) to our [pyOpenSci Fall Festival](events/pyopensci-2024-fall-festival.html)
+helpers at our workshops
+* if you'd like to get involved with pyOpenSci, check out our [volunteer page](/volunteer.html).
+
+You can also:
+
+* Keep an eye on our [events page](/events.html) for upcoming training events.
```

Comment:
```suggestion

So, what's next for pyOpenSci and these training programs? We're going to start some pilots! We'll be running a series of trianings that include a larger event, smaller workshops, and other training opportunities with various price points and sponsorships and scholarship models. Along the way, we want to hear from you! We know you want pyOpenSci to succeed too. What's working well, what could be better, where do you see opportunities to combine sustainability and training? 

As we always do, we'll have surveys as a part of each event, and create space to hear feedback and share ideas. We look forward to embarking on this training program path with you. 

### Our next event - Fall Festival

The first in this series is our [virtual Fall Festival](events/pyopensci-2024-fall-festival.html), October 28 - November 1. This event is a combination of free talks, paid workshops, community engagement, and different sponsorship and scholarships models to ensure broad participation. We've wrapped a lot together into one event!   

The pyOpenSci Fall Festival is an online community training and networking event designed to empower scientists with in-demand open science skills. Our goals for the Fall Festival are to:

* empower you with technically-relevant open science skills,
* call attention to and celebrate new and upcoming tools that support open reproducible science
* build community, and
* help raise funds for pyOpenSci operations that allow us to continue supporting community work.

We're so excited for this event, to starting to try these different approaches, and building and learning together with you. Find more information [on the event page](https://www.pyopensci.org/events/pyopensci-2024-fall-festival.html) and reach out to us if you have any questions. 

You can also keep an eye on our [events page](/events.html) for other upcoming training events.

I look forward to seeing you there! 
```

Reply:
Since this was "what's next for pyOpenSci and the road ahead" I broadened it out to talk about training more generally and then highlight the fall festival as the first event. 

I also emphasized the pilot aspect of it and how we're looking to try things and learn too.

Reply:
I like this! However, I will also do a bit of editing. It's technically our second online paid event. But I've also run a tutorial at SciPy and several sprints where I taught people Git and GitHub! 

Maybe I'll reframe this section regarding what types of events we've tried and where the FF sits in the grander scheme of training events and sustainability. Essentially, if we do more FF, it will support more sprints, online lessons, and other good things that have high impact.